docs/src/Fieldset.doc.js

```diff
@@ -249,6 +261,60 @@ function CheckboxExample() {
     </Flex>
   );
 }
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection title="Error message">
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+function CheckboxExample() {
+  const [favorite, setFavorite] = React.useState(null);
+
+  return (
+    <Flex direction="column" gap={4}>
+      <Flex direction="column" gap={2}>
+        <Heading size="sm">Do you like me</Heading>
+        <Text size="md">
+          Choose up to 3.
+          <Text inline size="md" weight="bold">
+            <Link inline target="blank" href="https://www.pinterest.com/">
+              Additional information
+            </Link>
+          </Text>
+        </Text>
+      </Flex>
+      <Fieldset id="fieldset-error-message" legend="Choose up to 3 company account goals" legendDisplay="hidden" errorMessage="Atleast 1 item must be selected">
+        <Flex direction="column" gap={4}>
+          <RadioButton
+            checked={favorite === 'pin'}
+            id="type-pin"
+            label="Pin"
```

Comment:
RadioButtons may not be ideal here since they can only have 1 selection by default.

Could you switch to checkboxes, and maybe have the legend say something like "Choose at least one sign in option", and then the error message can be "At least 1 item must be selected"